modules/appeals_api/spec/requests/v1/notifce_of_disagreements/contestable_issues_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../../support/shared_examples_contestable_issues'
```

Comment:
could we use `AppealsApi::Engine.root.join` vs the `../`'s would be more readable in my opinion

Reply:
Updated!